js/libs/form.js

```diff
@@ -114,6 +114,41 @@ Fliplet().then(function() {
       };
     }
 
+    function getMatrixValue(value, field) {
+      var matrixOption = {};
+
+      if (!value || typeof value === 'object') {
+        return value;
+      }
+
+      if (value.indexOf('[') > -1 || value.indexOf(']') > -1) {
+        _.forEach(value.split(/\r?\n/), function(rawOption) {
+          if (rawOption) {
+            rawOption = rawOption.trim();
+
+            var regex = /\[(.*)\]/g;
+            var match = rawOption.split(regex).filter(r => r !== '');
+
+            if (match.length > 1) {
+              matrixOption[match[0].trim()] =  match[1].trim();
```

Comment:
Use return to avoid using `else`, and therefore reduce the indentations.

```suggestion
              matrixOption[match[0].trim()] =  match[1].trim();
              return;
```